en_US.json

```diff
@@ -733,5 +735,37 @@
     "SPOOF_KD": "Spoof K/D Ratio",
     "SPOOF_BADSPORT": "Spoof Badsport State",
     "SPOOF_JOB_POINTS": "Spoof Job Points",
-    "SPOOF_PLAYER_MODEL": "Spoof Player Model"
+    "SPOOF_PLAYER_MODEL": "Spoof Player Model",
+    "OUTFIT_HEAD": "HEAD",
+    "OUTFIT_BERD": "BERD",
+    "OUTFIT_HAIR": "HAIR",
+    "OUTFIT_UPPR": "UPPR",
+    "OUTFIT_LOWR": "LOWR",
+    "OUTFIT_HAND": "HAND",
+    "OUTFIT_FEET": "FEET",
+    "OUTFIT_TEEF": "TEEF",
+    "OUTFIT_ACCS": "ACCS",
+    "OUTFIT_TASK": "TASK",
+    "OUTFIT_DECL": "DECL",
+    "OUTFIT_JBIB": "JBIB",
+    "OUTFIT_GLASSES": "GLASSES",
+    "OUTFIT_EARS": "EARS",
+    "OUTFIT_UNK1": "UNK1",
+    "OUTFIT_UNK2": "UNK2",
+    "OUTFIT_UNK3": "UNK3",
+    "OUTFIT_WATCH": "WATCH",
+    "OUTFIT_WRIST": "WRIST",
+    "OUTFIT_UNK4": "UNK4",
+    "OUTFIT_RANDOM_COMPONENT": "Random Component",
+    "OUTFIT_DEFAULT_COMPONENT": "Default Component",
+    "OUTFIT_RANDOM_PROPS": "Random Props",
+    "OUTFIT_CLEAR_PROPS": "Clear Props",
+    "EXPORT_TO_CLIPBOARD": "Export To Clipboard",
+    "IMPORT_FROM_CLIPBOARD": "Import From Clipboard",
+    " OUTFIT_TEX": " TEX",
```

Comment:
Weird space at start of translation label.

Reply:
Why in the label though? Please format the code differently on YimMenu so it works properly without this space in the label.

Use `std::vformat` or some different kind of string concatenation.